# Comments on last rotation
> Communication with others
pswd: upload123
pswd2: lulab123

ssh -p 10358 john@166.111.156.58

> passwd : command to reset password

ssh xxx@172.22.220.58: for inlab log-in
> shortcuts
> alias
> top  show which user and how much CPU left
# Pt II. Basic Bioinfo Analysis
## Compute on the server: HOW
> tmux, screen, nohub
## A real case
_exRNA_ exists extracellularly, in blood. 
###Backgound
Canonical ncRNAs
    - snRNA: Aid slice
    - snoRNA: Aid 
    - srpRNA: Signal recognition

Small ncRNAs
    - miRNA: 
    - piRNA:
    - siRNA:
Long ncRNA
    - MALAT1
    - HOTAIR

Group I ribozyme : A intron

200 copies of rrn genes in the genome(as DNA)

In one cell, rRNA >> mRNA ~ 10^6, Protein 10^9~10.

Small RNA has haiprin
pri -> pre


##RNA-seq
asdf

|Type   |Note  |
| ------|:--:|
|Default| whole cell polyA RNA-seq, > 200nt:|
|small| RNA-seq <50 nt|
|total| rRNA removed, >200nt|
|nuc.|  |
|exRNA-seq| |
asdf


###exRNA-seq 
cell free/ MV/ exosom /RNP; short/long
Usually we use polyA + to build library. 
By default small RNA-seq has strand information. 

Data are: __of high variance; Batch effect; Dropouts a lot__


#OK we are done with BKGD. Back to gitbook. 

Q 50-200nt RNA
A medium-sized



====================================================

#WANG, Siqi

Preprocessing -> Mapping -> Expression analysis

## Basid: RNA-seq library

R Sync

Scp

Transmit

fetch 
